docs/cn/MatrixOne/Get-Started/update-standalone-matrixone.md

```diff
@@ -0,0 +1,209 @@
+# **升级单机版 MatrixOne**
+
+MatrixOne 可以通过以下三种方法进行升级：
```

Comment:
Add description about this is only about in-0.5 version update. Not for 0.4/0.3/0.2/0.1 -> 0.5 version update.